spec/tools/miq_config_sssd_ldap/configure_appliance_settings_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require "configure_appliance_settings"
 
-describe MiqConfigSssdLdap::ConfigureApplianceSettings do
```

Comment:
Should this change be included in this PR?

Reply:
rspec 4 is going to make the outer describe mandatory, so I've been doing this as I go so we don't have to do it later.

Reply:
Can you make a separate PR for that so that the whole repo is consistent?
There are current 66 lines of `RSpec.describe SomeClass` and 497 lines of just `describe SomeClass` (without the `RSpec`).

Reply:
I'll do one soon, sure.

Reply:
@bdunne But I would ask that you not hold up this particular PR over it.